addons/website/views/snippets/snippets.xml

```diff
@@ -219,8 +219,9 @@
 
     <div data-js="layout_column"
         data-selector="section"
-        data-target="> * > .row:not(.s_nb_column_fixed)">
+        data-target="> *:has(.row:not(.s_nb_column_fixed)), > .o_we_zero_cols">
```

Comment:
- Are you not missing a `>` before `.row`, like before ?
- I find the name o_we_zero_cols confusing (since there are containers with 0 cols that do not have that class), could not it be something like allow_columns or something like that?

Reply:
Considering that at the moment it's only used to allow adding column where there are none, that seems like a better name. The spec was initially framed as allowing to remove columns where they already existed so that name made more sense then, I'll change it :)